tsdb/points.go

```diff
@@ -488,8 +488,9 @@ func isNumeric(b byte) bool {
 // scanNumber returns the end position within buf, start at i after
 // scanning over buf for an integer, or float.  It returns an
 // error if a invalid number is scanned.
-func scanNumber(buf []byte, i int) (int, []byte, error) {
+func scanNumber(buf []byte, i int) (int, error) {
```

Comment:
I made this change as the `[]byte` of the return was ignored and only called in one spot.